front/app/containers/Authentication/useSteps/stepConfig/sharedSteps.ts

```diff
@@ -167,6 +167,9 @@ export const sharedSteps = (
         if (error_code === 'not_entitled_under_minimum_age') {
           setCurrentStep('missing-data:verification');
```

Comment:
Should we also change this to `access-denied`?

Reply:
I tried that, but it didn't make sense as this is also what gets returned from standalone verification where you cannot change the error message. So thought it best to leave it like this for now.

Reply:
> as this is also what gets returned from standalone verification where you cannot change the error message.

Not sure I follow, why can't the standalone verification also use the same `access-denied` step with a custom error message?